Mobile/stockdogmobile/components/widebutton.js

```diff
@@ -1,48 +1,47 @@
 import React, { Component } from 'react';
 import { Text, TouchableOpacity } from 'react-native';
-import elements from '../style/elements';
-import text from '../style/text';
+import styles from '../style/components/widebutton';
 
 export default class WideButton extends Component {
-  constructor(props) {
-    super(props);
-  }
+   constructor(props) {
+      super(props);
+   }
 
-  render() {
-    var buttonText = "";
-    if (this.props.type == 'login')
-      buttonText = 'LOGIN';
-    else if (this.props.type == 'register')
-      buttonText = 'REGISTER';
-    else if (this.props.type == 'buy')
-      buttonText = 'BUY';
-    else if (this.props.type == 'sell')
-      buttonText = 'SELL';
-    else if (this.props.type == 'portfolio')
-      buttonText = 'Next';
-    else if (this.props.type == 'join')
-      buttonText = 'Join league!';
-    else if (this.props.type == 'cancel')
-      buttonText = 'CANCEL';
-    else if (this.props.type == 'logout')
-      buttonText = 'LOGOUT';
+   render() {
+      var buttonText = "";
+      if (this.props.type == 'login')
```

Comment:
Minor suggestion of using switch statement instead of if else for readability

Reply:
Agreed! Just changed